avocado/core/safeloader.py

```diff
@@ -27,18 +27,37 @@
 from ..utils import data_structures
 
 
-class AvocadoModule:
+class PythonModule:
     """
-    Representation of a module that might contain avocado.Test tests
+    Representation of a Python module that might contain interesting classes
+
+    By default, it uses module and class names that matches Avocado
+    instrumented tests, but it's supposed to be agnostic enough to
+    be used for, say, Python unittests.
     """
-    __slots__ = 'path', 'test_imports', 'mod_imports', 'mod', 'imported_objects'
+    __slots__ = ('path', 'test_imports', 'mod_imports', 'mod', 'imported_objects',
+                 'module', 'klass')
 
-    def __init__(self, path):
+    def __init__(self, path, module='avocado', klass='Test'):
+        """
+        Instantiates a new PythonModule representation
+
+        :param path: path to a Python source code file
+        :type path: str
+        :param module: the original module name from where the
+                       possibly interesting class must have been
+                       imported from
+        :type module: str
+        :param module: the possibly interesting class original name
```

Comment:
should be `:param klass`

Reply:
@clebergnu only doc string  minor change , other wise looks good

Reply:
@PraveenPenguin good catch, thanks!